scrapy/utils/py36.py

```diff
@@ -1,10 +0,0 @@
-"""
```

Comment:
Don't we want to deprecate this first? https://github.com/scrapy/scrapy/pull/4467/files#diff-0ec586c8bb4f8b18d1add1d954eb5448